src/kaleidoscope/hid.cpp

```diff
@@ -135,8 +135,13 @@ void initializeMouse() {
 
 void moveMouse(signed char x, signed char y, signed char wheel) {
   Mouse.move(x, y, wheel);
+  //moveMouseWithHorizontalWheel(x, y, wheel, 0);
 }
 
+void moveMouseWithHorizontalWheel(signed char x, signed char y, signed char vWheel, signed char hWheel) {
```

Comment:
I believe we could combine `moveMouse` and `moveMouseWithHorizontalWheel`, like this:

```c++
void moveMouse(signed char x, signed char y, signed char vWheel = 0, signed char hWheel = 0) {
  Mouse.move(x, y, vWheel, hWheel);
}
```

With the default values for the `vWheel` and `hWheel` arguments, `moveMouse(x, y)` and `moveMouse(x, y, vWheel)` will continue working as before.

No API breakage, no new functions introduced!